tensorflow_lite_support/ios/task/processor/sources/TFLDetectionResult.h

```diff
@@ -26,18 +26,39 @@ NS_ASSUME_NONNULL_BEGIN
  * The index of the image classifier head these classes refer to. This is useful for multi-head
  * models.
  */
-@property(nonatomic, assign) CGRect boundingBox;
+@property(nonatomic, assign, readonly) CGRect boundingBox;
 
 /** The array of predicted classes, usually sorted by descending scores (e.g.from high to low
  * probability). */
-@property(nonatomic, copy) NSArray<TFLCategory *> *categories;
+@property(nonatomic, copy, readonly) NSArray<TFLCategory *> *categories;
```

Comment:
since you're dealing with the synthesized variable directly rather than using the property in the implementation, it's fine to remove `copy` here as you're not using it.

Reply:
Removed